test/modules/map/components/contextMenu/MapContextMenuContent.test.js

```diff
@@ -6,8 +6,10 @@ import { notificationReducer } from '../../../../../src/store/notifications/noti
 import { Icon } from '../../../../../src/modules/commons/components/icon/Icon';
 import { LevelTypes } from '../../../../../src/store/notifications/notifications.action';
 import { GlobalCoordinateRepresentations } from '../../../../../src/domain/coordinateRepresentation';
+import { SharePositionChip } from '../../../../../src/modules/map/components/assistChips/SharePositionChip';
 
 window.customElements.define(MapContextMenuContent.tag, MapContextMenuContent);
+window.customElements.define(SharePositionChip.tag, SharePositionChip);
```

Comment:
Let's test it just with a shallow element, no need for importing the `SharePositionChip` component